segments/router.py

```diff
@@ -20,19 +20,5 @@ def db_for_read(self, model, **hints):
     def db_for_write(self, model, **hints):
         return None
 
-    def allow_relation(self, obj1, obj2, **hints):
-        """
-        If we're trying to create a relationship between two objects, one of which is a model
-        from the Segments app, and at least one of which uses SEGMENTS_EXEC_CONNECTION, then allow it.
-        Namely this targets Segment.refresh() where something like this happens:
-
-            SegmentMembership(segment=s, user=u)
-
-        Where 'u' is coming from a SEGMENTS_EXEC_CONNECTION queryset.
-        """
-        is_segments = obj1._meta.app_label == 'segments'
-        is_conn = obj2._state.db == SEGMENTS_EXEC_CONNECTION
-        return True if is_segments and is_conn else None
-
```

Comment:
Do we need the router at all ? Looks like it does not have any logic, I think we can remove it